### ISSUE TITLE
Enhancement: Adding a different thread

### ISSUE THREAD

**Issue by mvp-2003** [Ash515/AutomaDesk#21]
In the #12, I mentioned about an idea of **Downloads Router**.
This issue is an _enhancement_ to the project.

Since this is a _GUI application_, it will be a **heavy software** to keep running in the background. But for _Automation_, the software needs to be kept running in the background. So, it will be better if we create a _different_ **thread** that can be _started_ and _stopped_ by the GUI app. This sole purpose of the thread will be doing the automations that the user chooses for in the GUI app. This thread can also be stopped by the user when not needed.

- [ ] A GUI that let's the user choose the automations available.

- [ ]  A thread running in the background that does the actual work.

**Note:**  Not sure how to achieve this and would need a team to accomplish this.

**Comment by Tankala-Sai-Lahari** [comment]
I would like to contribute this , I have some idea please assign it to me

**Comment by Tankala-Sai-Lahari** [comment]
@mvp-2003

**Comment by Tankala-Sai-Lahari** [comment]
@Ash515

**Comment by mvp-2003** [comment]
Yes, that is also my idea. I am currently working on the Downloads router which is taking longer than expected. I would like to have a frontend for this but the problem is that when we integrate the code, things will get messy. So we would need a planned frontend first.
Also, the sign-in feature doesn't seem necessary.

**Comment by mvp-2003** [comment]
ok. Yeah but a frontend should be done soon
